loadtest/scripts/populate_genesis_accounts.py

```diff
@@ -34,45 +43,106 @@ def add_genesis_account(account_name, lock, local=False):
             "mnemonic": mnemonic,
         }
         json.dump(data, f)
-    success = False
-    retry_counter = 5
+
+    return add_account_cmd
+
+
+def create_genesis_account(account_index, account_name, local=False):
+    address, mnemonic = add_key(account_name=account_name, local=local)
+    add_account_cmd = add_account(account_name=account_name, address=address, mnemonic=mnemonic, local=local)
+
+    retry_counter = 0
     sleep_time = 1
-    while not success and retry_counter > 0:
+
+    while True:
         try:
-            with lock:
-                subprocess.check_call(
-                    [add_account_cmd],
-                    shell=True,
-                    timeout=20,
-                )
-                success = True
+            print(f'Running: ${add_account_cmd}')
+            subprocess.call(
+                [add_account_cmd],
+                shell=True,
+                timeout=20,
+            )
+            break
         except subprocess.CalledProcessError as e:
-            print(f"Encountered error {e}, retrying {retry_counter - 1} times")
-            retry_counter -= 1
+            print(f"Encountered error {e}, retried {retry_counter} times")
+            retry_counter += 1
             sleep_time += 0.5
             time.sleep(sleep_time)
 
+    global_accounts_mapping[account_index] = {
+        "balance": {
```

Comment:
took these two values directly from the output of a genesis.json file from one of the load test nodes, other than the address everything was the same